<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1514070920288' stop='1514070927481'>
    <name>Testing the report and log</name>
    <title>Testing the report and log</title>
    <test-cases>
        <test-case start='1514070920289' status='failed' stop='1514070924629'>
            <name>validating correct title</name>
            <title>validating correct title</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>TypeError: browser.takeScreenshot(...).afterEach is not a function</message>
                <stack-trace>TypeError: browser.takeScreenshot(...).afterEach is not a function
                    at UserContext.&lt;anonymous> (/Users/suyashnande/WebstormProjects/Protractor_Test/conf.js:29:38)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                    at /usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4264:13
                    at /usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4172:9
                    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:64:48
                    at ControlFlow.emit (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/events.js:62:21)
                    at ControlFlow.shutdown_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2674:10)
                    at shutdownTask_.MicroTask (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2599:53)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1514070924629' status='failed' stop='1514070927481'>
            <name>validating in-correct title</name>
            <title>validating in-correct title</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Expected 'Protractor practice website - Banking App' to contain 'ProtractorOne'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (/Users/suyashnande/WebstormProjects/Protractor_Test/validate_spec.js:20:23)
                    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:112:25
                    at new ManagedPromise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
                    at ControlFlow.promise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
                    at schedulerExecute (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:95:18)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2974:25)
                    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>